draft-jasdips-regext-rdap-rpki.md

```diff
@@ -214,8 +220,9 @@ Searches for ROA information by origin autonomous system number are specified us
 
 rpkiRoas?originAutnum=AAAA
 
-AAAA is an autonomous system number representing the "originAutnum" property of a ROA, as described in (#roa_object_class).
-The following URL would be used to find information for ROAs with origin autonomous system number 65536:
+AAAA is an autonomous system number representing the "originAutnum" property of a ROA, as described in
```

Comment:
AAAA is also the IPv6 DNS address record. Maybe just "AA" or use "XXX" as RFC 9082 does. Just a suggestion... not a big deal.

Reply:
> AAAA is also the IPv6 DNS address record. Maybe just "AA" or use "XXX" as RFC 9082 does. Just a suggestion... not a big deal.

Good point. Easy enough to use another literal.